BAZProjectC1/Components/BasicInfoMovie/backButton.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  backButton.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 28/09/22.
+//
+import UIKit
+
+class backButton: UIButton {
```

Comment:
Recuerda la convención de nombrado de clases
```suggestion
class BackButton: UIButton {
```